api/repo.go

```diff
@@ -359,6 +359,11 @@ func GetRepo(c *gin.Context) {
 	c.JSON(http.StatusOK, r)
 }
 
+func ffff(c *gin.Context) {
```

Comment:
Is this something you wanted to commit? Can we delete it?

Reply:
That was uploaded by accident. I have removed it. Thanks for spotting it!